azure-specialized-workloads/sap/recommendations.yaml

```diff
@@ -349,12 +349,12 @@
     - name: ASCS-Pacemaker - Central Server Instance
       url: "https://docs.microsoft.com/en-us/azure/advisor/advisor-reference-reliability-recommendations"
 
-- description: ASCS-LB (Central Server Instance) Ensure the load balancer is configured correctly for SAP ASCS High availability
+- description: Ensure the central server load balancer is configured correctly for SAP ASCS High availability
```

Comment:
This should say - "Ensure the  load balancer is configured correctly for SAP ASCS High availability"